src/OpenLoco/src/Windows/News/News.cpp

```diff
@@ -458,11 +459,40 @@ namespace OpenLoco::Ui::Windows::NewsWindow
         // 0x0042A136
         static void sub_42A136(Window* self, Gfx::RenderTarget* rt, Message* news)
```

Comment:
Perhaps we can rename this to drawNewsSubjects?

Reply:
hmm its more like drawNonViewportNewsSubjects

Reply:
Hmm, that's a bit long. I suppose `drawOtherNewsSubjects` is too vague?

Reply:
drawNewsSubjectImages ? would that be better?

Reply:
Yeah, sounds good to me!